bin/cli.js

```diff
@@ -136,8 +164,12 @@ function invoke(env) {
       'Specify the stub extension (default js)'
     )
     .action(function(name) {
-      var instance = initKnex(env);
-      var ext = (argv.x || env.configPath.split('.').pop()).toLowerCase();
+      var instance = initKnex(env, commander);
```

Comment:
do we need to pass commander instance for internal calls? can't we pass resolved params instead?

Reply:
There is a method to do that on commander API? Can't find.